src/js/views.js

```diff
@@ -170,31 +207,60 @@ window.RegionDetailsView = Backbone.View.extend({
 
     render: function() {
         var self = this;
-
         this.$el.html(this.template(this.model.attributes));
 
         this.$el.find('#subregion-list').html(this.model.get('subregions').map(function(subregion) {
-            return new window.SubregionDetailsView({ model: subregion }).render().el;
+            var view = new window.SubregionDetailsView({ model: subregion });
+
+            self.listenTo(view, 'toggle', self.toggleSubregionDetails);
```

Comment:
Does this listener go out of scope when the view's destroyed?

Reply:
I read through some things (including this: https://github.com/jashkenas/backbone/issues/828), and I think we are in the clear. The listeners appear to get garbage collected when the DOM elements are removed. But good catch.